Godnoken/Sprint-Challenge-Applied-Javascript
# Sprint Challenge: Applied JavaScript - Lambda School

## What does this PR do?
Sprint Challenge: Applied JavaScript I MVP and start of Stretch goal
​
## Description of tasks completed
- Answered all the self-study questions in README.md
- Implemented a header component and tab + cards components
- Pulled data with axios.get to give values to components
- (Stretch Goal) Implemented carousel component
​
## How can this be tested?
​Clone the project.
Open up the project files in your favourite code editor.
Run index.html to see components and axios.get in action.
Check the index.js files and carousel.js files for code.
​
## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/68021050-ad103c00-fca0-11e9-821a-277eefd7ed5d.png)
